internal/db/queryBuilder.go

```diff
@@ -111,50 +118,66 @@ func (qb *queryBuilder) removedObjectsCTE() string {
 	return fmt.Sprintf(template, ignoresPredicate)
 }
 
-func (qb *queryBuilder) queryWithoutRemovals() string {
-	template := `
-	WITH possible_objects AS (
-	%s
-	), updated_objects AS (
-	%s
-	)
-
-	SELECT path, mode, size, bytes, packed, deleted
-	FROM updated_objects
+func (qb *queryBuilder) selectWithoutRemovals() string {
+	return `
+		SELECT path, mode, size, bytes, packed, deleted, (hash).h1, (hash).h2
+		FROM updated_objects
 	`
-
-	return fmt.Sprintf(template, qb.possibleObjectsCTE(), qb.updatedObjectsCTE())
 }
 
-func (qb *queryBuilder) queryWithRemovals() string {
+func (qb *queryBuilder) queryWithoutRemovals(asCte bool) string {
 	template := `
-	WITH possible_objects AS (
-	%s
-	), updated_objects AS (
-	%s
-	), removed_objects AS (
-	%s
-	)
-
```

Comment:
I removed the space between the CTE and select statement. Was that important for pganalyze stats?